asciidoc-extensions/src/main/java/org/beanvalidation/asciidocextensions/BeanValidationAsciidoctorExtensionRegistry.java

```diff
@@ -0,0 +1,22 @@
+package org.beanvalidation.asciidocextensions;
```

Comment:
Could you add a license header (same for some other files, e.g. the Ruby class).